docker/docker-compose-dev.yml

```diff
@@ -110,11 +110,12 @@ services:
   dashboard_worker:
     build: *dashboard-build
     environment: *dashboard-environment
-    command: celery -A dashboard_viewer worker -Ofair -l INFO
+    command: celery -A dashboard_viewer worker -Ofair -l DEBUG
```

Comment:
@aspedrosa DEBUG?

Reply:
This is the development docker-compose file. Note the file name.